egs2/hifitts/tts1/run.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+fs=24000
+n_fft=2048
+n_shift=300
+win_length=1200
+
+opts=
+if [ "${fs}" -eq 48000 ]; then
+    # To suppress recreation, specify wav format
+    opts="--audio_format wav "
+else
+    opts="--audio_format flac "
+fi
```

Comment:
I saw that in the readme you said the recipe is for 22.05kHz setup, but the default fs is set to 24000 and 48000 (for raw waveform)? which seems to be a conflict